web/src/app/schedules/calendar/ScheduleCalendar.tsx

```diff
@@ -268,14 +336,15 @@ function ScheduleCalendar(props) {
               fontFamily: theme.typography.body2.fontFamily,
               fontSize: theme.typography.body2.fontSize,
             }}
-            tooltipAccessor={() => null}
+            tooltipAccessor={() => ''}
```

Comment:
I can't find good docs for what this param is needed for, but it usually takes return type of string so I'm doing this. Should we remove this parameter or have it return something else?